src/modules/planetside/CensusApi.ts

```diff
@@ -141,6 +141,30 @@ class CensusApi {
     }
   }
 
+  async getCharacterCerts(name: string) {
```

Comment:
Certs are already a part of the base Character returned from the API, even though I might have forgotten to include them in the Character type. No need to use a custom Item

Reply:
Does that mean I should modify the character type and the getDetailedCharacterByName function?

Reply:
Just create a simple general getCharacter method, similar to getOutfit.